check-code-style/check_style_of_all_files.sh

```diff
@@ -4,9 +4,9 @@ source $(dirname "$0")/check_style.sh
 
 # Check every file for correct code style.
 check_style_of_all_files() {
-  # Find all files we want to check in the workspace.
+  # Find all files we want to check excluding files in the 'submodules' directories.
```

Comment:
Please reference comment above (as in, specify the file name) and also add a reference to the comment above into this file as well so that anyone taking this one will know to update both files. Thanks!